timed/employment/models.py

```diff
@@ -39,6 +39,28 @@ class Employment(models.Model):
     start_date       = models.DateField()
     end_date         = models.DateField(blank=True, null=True)
 
+    @classmethod
+    def employment_at(cls, user, date):
```

Comment:
the preferred way to add table-level functionality is model Managers - see https://docs.djangoproject.com/en/1.11/topics/db/managers/#adding-extra-manager-methods